DQM/L1TMonitor/src/L1TStage2uGT.cc

```diff
@@ -15,18 +15,35 @@
 L1TStage2uGT::L1TStage2uGT(const edm::ParameterSet& params):
    l1tStage2uGtSource_(consumes<GlobalAlgBlkBxCollection>(params.getParameter<edm::InputTag>("l1tStage2uGtSource"))),
    monitorDir_(params.getUntrackedParameter<std::string> ("monitorDir", "")),
-   verbose_(params.getUntrackedParameter<bool>("verbose", false))
+   verbose_(params.getUntrackedParameter<bool>("verbose", false)),
```

Comment:
Construct the gtUtils_ here and erase the five lines in the constructor implementation.
`gtUtil_(new l1t::L1TGlobalUtil(params, consumesCollector(), *this, params.getParameter<edm::InputTag>("l1tStage2uGtSource"), params.getParameter<edm::InputTag>("l1tStage2uGtSource"))),`